src/model/PartyUser.ts

```diff
@@ -49,7 +51,9 @@ export type PartyUserOnCreation = {
   email: string;
   confirmEmail: string;
   productRoles: Array<string>;
-  certification: boolean;
+  certifiedName: boolean;
+  certifiedSurname: boolean;
+  certifiedMail: boolean;
 };
```

Comment:
Are these already existing?
```suggestion
};
```
```suggestion
  certifiedName: boolean;
  certifiedSurname: boolean;
  certifiedMail: boolean;
};
```